crates/sui-core/src/consensus_handler.rs

```diff
@@ -133,6 +132,19 @@ impl<T: ParentSync + Send + Sync> ExecutionState for ConsensusHandler<T> {
         let mut transactions = vec![];
         let timestamp = consensus_output.sub_dag.commit_timestamp;
 
+        let epoch_start = self
+            .epoch_store
+            .epoch_start_config()
+            .epoch_start_timestamp_ms();
+        let timestamp = if timestamp < epoch_start {
+            error!(
+                "Unexpected commit timestamp {timestamp} less then epoch start time {epoch_start}"
```

Comment:
It would be nice to have a few more info of the leader here (ex round, author )

Reply:
https://github.com/MystenLabs/sui/pull/10394